pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -176,7 +178,8 @@ func PostStartHook(hookContext genericapiserver.PostStartHookContext) error {
 		}
 
 		// ensure bootstrap rolebindings are created or reconciled
-		for _, clusterRoleBinding := range append(bootstrappolicy.ClusterRoleBindings(), bootstrappolicy.ControllerRoleBindings()...) {
+		bindNodeRole := !utilfeature.DefaultFeatureGate.Enabled(features.NodeAuthorizer)
+		for _, clusterRoleBinding := range append(bootstrappolicy.ClusterRoleBindings(bindNodeRole), bootstrappolicy.ControllerRoleBindings()...) {
```

Comment:
switched to a filter that gets enabled when using the node authorizer. it's ugly, but will go away in 1.8. try not to fixate on it.